features/environment.py

```diff
@@ -42,12 +43,13 @@ def before_all(context):
     context.base_url = "https://bortnik.hipchat.com"
     context.driver = webdriver.Chrome()
     context.wait = ui.WebDriverWait(context.driver, 10)
-
     context.login_page = LoginPage(context)
     context.authorized_page = AuthorizedPage(context)
     context.lobby_page = LobbyPage(context)
     context.api_page = ApiPage(context)
     context.settings_page = SettingsPage(context)
+    context.people_page = PeoplePage(context)
+    context.wait = ui.WebDriverWait(context.driver, 10)
```

Comment:
I think at line 45 we got somethink similar:)

Reply:
Done